src/test/java/org/zephyrsoft/sdb2/model/XMLConverterTest.java

```diff
@@ -35,128 +35,65 @@
-		"    <setting>\n" +
-		"        <key>BACKGROUND_COLOR</key>\n" +
-		"        <value class=\"awt-color\">\n" +
-		"            <red>1</red>\n" +
-		"            <green>2</green>\n" +
-		"            <blue>3</blue>\n" +
-		"            <alpha>255</alpha>\n" +
-		"        </value>\n" +
-		"    </setting>\n" +
-		"    <setting>\n" +
-		"        <key>BOTTOM_MARGIN</key>\n" +
-		"        <value class=\"int\">25</value>\n" +
-		"    </setting>\n" +
-		"    <setting>\n" +
-		"        <key>COPYRIGHT_FONT</key>\n" +
-		"        <value class=\"awt-font\">\n" +
-		"            <name>Dialog.bold</name>\n" +
-		"            <style>1</style>\n" +
-		"            <size>14</size>\n" +
-		"        </value>\n" +
-		"    </setting>\n" +
-		"    <setting>\n" +
-		"        <key>LOGO_FILE</key>\n" +
-		"        <value class=\"string\">/home/mathis/Bilder-Incoming/20170917-1927_img_0155.jpg</value>\n" +
-		"    </setting>\n" +
-		"    <setting>\n" +
-		"        <key>SCREEN_2_CONTENTS</key>\n" +
-		"        <value class=\"org.zephyrsoft.sdb2.model.ScreenContentsEnum\">OnlyLyrics</value>\n" +
-		"    </setting>\n" +
-		"    <setting>\n" +
-		"        <key>SHOW_TITLE</key>\n" +
-		"        <value class=\"boolean\">false</value>\n" +
-		"    </setting>\n" +
-		"    <setting>\n" +
-		"        <key>SONG_LIST_FILTER</key>\n" +
-		"        <value class=\"org.zephyrsoft.sdb2.model.FilterTypeEnum\">TitleAndLyrics</value>\n" +
-		"    </setting>\n" +
-		"</settings>\n";
+	private static final String SETTINGS_XML = "<?xml version=\"1.0\" encoding=\"UTF-8\" standalone=\"yes\"?>\n"
+		+ "<settings>\n" + "    <setting>\n" + "        <key>BACKGROUND_COLOR</key>\n"
+		+ "        <value class=\"awt-color\">\n" + "            <red>1</red>\n" + "            <green>2</green>\n"
+		+ "            <blue>3</blue>\n" + "            <alpha>255</alpha>\n" + "        </value>\n"
+		+ "    </setting>\n" + "    <setting>\n" + "        <key>BOTTOM_MARGIN</key>\n"
+		+ "        <value class=\"int\">25</value>\n" + "    </setting>\n" + "    <setting>\n"
+		+ "        <key>COPYRIGHT_FONT</key>\n" + "        <value class=\"awt-font\">\n"
+		+ "            <name>Dialog.bold</name>\n" + "            <style>1</style>\n"
+		+ "            <size>14</size>\n" + "        </value>\n" + "    </setting>\n" + "    <setting>\n"
+		+ "        <key>LOGO_FILE</key>\n"
+		+ "        <value class=\"string\">/home/mathis/Bilder-Incoming/20170917-1927_img_0155.jpg</value>\n"
+		+ "    </setting>\n" + "    <setting>\n" + "        <key>SCREEN_2_CONTENTS</key>\n"
+		+ "        <value class=\"org.zephyrsoft.sdb2.model.ScreenContentsEnum\">OnlyLyrics</value>\n"
+		+ "    </setting>\n" + "    <setting>\n" + "        <key>SHOW_TITLE</key>\n"
+		+ "        <value class=\"boolean\">false</value>\n" + "    </setting>\n" + "    <setting>\n"
+		+ "        <key>SONG_LIST_FILTER</key>\n"
+		+ "        <value class=\"org.zephyrsoft.sdb2.model.FilterTypeEnum\">TitleAndLyrics</value>\n"
+		+ "    </setting>\n" + "</settings>\n";
```

Comment:
I liked the code formatting better before, please restore it. It's much more readable the old way. You are free to add comments around this block to switch off the formatting of your IDE if that helps.

Reply:
just a mistake. Sorry for that.